Koha/Plugin/EDS/bootstrap/modules/eds-detail.tt

```diff
@@ -43,14 +43,21 @@
 [% INCLUDE 'masthead.inc' %]
 
 <div class="main">
-    <ul class="breadcrumb">
-        <li><a href="/cgi-bin/koha/opac-main.pl">[% EDSLANG.home_breadcrumb %]</a> <span class="divider">&rsaquo;</span></li>
-        <li><a href="#">[% EDSLANG.eds_detail_TT_details_for %] [% title |html %][% FOREACH subtitl IN subtitle %], [% subtitl.subfield |html %][% END %]</a></li>
-    </ul>
+ <nav aria-label="breadcrumb">
+        <ul class="breadcrumb">
+            <li class="breadcrumb-item">
+                <a href="/cgi-bin/koha/opac-main.pl">Home</a>
+            </li>
+            <li class="breadcrumb-item" aria-current="page">
+                <a href="#"><span>Details for: </span> [% title |html %][% FOREACH subtitl IN subtitle %], [% subtitl.subfield |html %][% END %]</a>
```

Comment:
This looks like another case where the language file template has been overwritten with hardcoded "Details for:"

Reply:
Review comments fixed.